src/PIL/Image.py

```diff
@@ -97,7 +97,7 @@ class DecompressionBombError(Exception):
     # import Image and use the Image.core variable instead.
     # Also note that Image.core is not a publicly documented interface,
     # and should be considered private and subject to change.
-    from . import _imaging as core
+    from .. import _imaging as core
```

Comment:
This change isn't related to your initial description. Was it unintentional?

Reply:
Oh,Sorry. I didn't delete it after the test.

Reply:
Thanks